<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>591</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="snapTab">
   <attribute name="title">
    <string>Snap</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="snpCB" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="snpCB1">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <item>
          <property name="text">
           <string>Camera 0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="snpCB2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <item>
          <property name="text">
           <string>Camera 0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="snpCWgt" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="snpPlaybtn">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="snpStopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="snpCBT">
         <property name="maxCount">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>Defect</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Perfect</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NewType</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="snpSnapBtn">
         <property name="text">
          <string>Snap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="snpShowWgt" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="snpLbl1">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="snpLbl2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="fdTab">
   <attribute name="title">
    <string>File Detect</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="fdWgt" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="fdfLE"/>
       </item>
       <item>
        <widget class="QPushButton" name="fdfBtn">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fdnLE"/>
       </item>
       <item>
        <widget class="QPushButton" name="fdnBtn">
         <property name="text">
          <string>Net</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="fdWgt2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="fdRst">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fddBtn">
         <property name="text">
          <string>Detect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fdLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="rtdTab">
   <attribute name="title">
    <string>RT Detect</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="rtdCB" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="rtdCB1">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <item>
          <property name="text">
           <string>Camera 0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rtdCB2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <item>
          <property name="text">
           <string>Camera 0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rtdCWgt" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="rtdLE1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rtdNet1Btn">
         <property name="text">
          <string>Net1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rtdLE2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rtdNet2Btn">
         <property name="text">
          <string>Net2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="rtdPlayBtn">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rtdDtctBtn">
         <property name="text">
          <string>Detect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rtdStopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rtdShowWgt" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="rtdLbl1">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rtdLbl2">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="stgTab">
   <attribute name="title">
    <string>Settings</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stgRmdLbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Path To Save</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stgRcvBtn">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Set as default</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stgImgLbll">
         <property name="text">
          <string>Images:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stgImgLine">
         <property name="text">
          <string>./pic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stgImgBtn">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stgVduLbl">
         <property name="text">
          <string>Videos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stgVduLine">
         <property name="text">
          <string>./vdu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stgVduBtn">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="ihTab">
   <attribute name="title">
    <string>Info</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
